### PR TITLE
services/notifications: Use new `content` array for each service instance

### LINKED ISSUE #277: `content` array is leaking between tests
The `content` array in the `notifications` service is currently shared between all instances of the service, which means that the contents of this array are also shared between tests. If I use `autoClear = false` and add a notication in one test, then that notification will also be visible in all following tests.

This can be avoided by assigning `content` in the `init()` hook instead.

### PR DESCRIPTION
**Pull request by Turbo87**
Resolves #277

/cc @mansona

### CODE REVIEW

File: addon/services/notifications.js [hunk 1]
```diff
@@ -11,7 +11,10 @@ const notificationAssign = assign || merge;
 const globals = config['ember-cli-notifications'] || {}; // Import app config object
 
 export default Service.extend({
-  content: A(),
+  init() {
+    this._super(...arguments);
+    this.set('content', A());
+  }
```
Comment (mansona) [inline addon/services/notifications.js]:
```suggestion
  },
```

**Review by mansona — CHANGES_REQUESTED** [review]

File: addon/services/notifications.js (hunk 1 above)
Comment (mansona) [reply addon/services/notifications.js]: 🙃  😂

Comment (Turbo87) [reply addon/services/notifications.js]: lol, that's what I get for copy pasting from a ES6 class :D

**Review by mansona — APPROVED** [review]
looks good thanks 🎉

**Comment by mansona** [comment]
I'm not entirely sure why Travis reported twice 🤔 

🤷 well I'll merge it anyway

**Pull request merged**